generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/TemporalFieldValueSource.java

```diff
@@ -15,7 +15,7 @@
 
 public class TemporalFieldValueSource implements FieldValueSource {
 
-    public static final LocalDateTime ISO_MAX_DATE = LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999);
+    public static final LocalDateTime ISO_MAX_DATE = LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999_000_000);
```

Comment:
why `999_000_000` and not `999_999_999`?

Reply:
`999_999_999` is more correct, however we increment by millisecond rather than nano second, therefore the last representable time is `999_000_000`. When we implement temporal granularity this may change.